crates/sncast/tests/e2e/account/delete.rs

```diff
@@ -119,7 +113,7 @@ pub fn test_happy_case() {
 }
 
 #[test]
-pub fn test_happy_case_without_network_args() {
+pub fn test_happy_case_url() {
```

Comment:
I don't see any difference between `test_happy_case_url` and `test_happy_case_without_network_args`. Seems they are duplicated/have missing args?